deployment/docker/README.md

```diff
@@ -21,6 +21,30 @@ password in front like this:
 ditto:A6BgmB8IEtPTs
 ```
 
+## Configuration of the services
+
+You may configure each service via passing variables to the java VM in the entrypoint section for each
+service.
+
+```yml
+    ...
+    entrypoint:
+      - java
+      # Alternative approach for configuration of the service
+      - -Dditto.gateway.authentication.devops.password=foobar
+      - -jar
+      - starter.jar
+```
+
+To get a list of available configuration options you may retrieve them from a running instance via:
+
+```bash
+# Substitute gateway with the service you are interested in
+curl http://devops:foobar@localhost:8080/devops/config/gateway/1?path=ditto
```

Comment:
The `1`  in the URL path is only valid for docker-swarm based deployments.
In general this is the `instanceIndex` to put in there - for kubernetes based deployments that would be the Pod-ID instead.